data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -26,8 +26,13 @@ for name in ironic ironic-inspector ironic-ramdisk-logs dnsmasq httpd coreos-dow
 done
 
 # Start the provisioning nic if not already started
-PROVISIONING_NIC={{.PlatformData.BareMetal.ProvisioningInterface}}
 
+#HACK
+#Search dmesg for the renamed interface for either eth0 or eth1
+#dmesg -t | grep 'renamed.*eth0'
+#igb 0003:02:00.0 eno1: renamed from eth0
+DMSG_PROVISIONING_NIC={{.PlatformData.BareMetal.ProvisioningInterface}}
+PROVISIONING_NIC=`dmesg -t  | grep 'renamed.*'$DMSG_PROVISIONING_NIC | sed s/://g | awk '{print $3}'`
```

Comment:
1 space too much after `-t` and let's make the grep as specific as possible. I think it also contains the word "from". Can you add that?